compiler/packages/babel-plugin-react-compiler/src/__tests__/fixtures/compiler/meta-isms/repro-cx-assigned-to-temporary.expect.md

```diff
@@ -53,22 +53,27 @@ import { identity } from "shared-runtime";
 const DARK = "dark";
 
 function Component() {
-  const $ = _c(2);
+  const $ = _c(4);
   const theme = useTheme();
 
-  const t0 = cx({
-    "styles/light": true,
-    "styles/dark": theme.getTheme() === DARK,
-  });
+  const t0 = theme.getTheme();
```

Comment:
i need to debug why the calls are getting reordered here, i suspect that the cx() call is simply getting inlined but i'm not sure